File: Python Unit/currency.py
#Practicing calculations with currency exchange
print("Let's exchange our trip money back into dollars!")
pesos = int(input('What do you have left in pesos? '))
reais = int(input('What do you have left in reais? '))
soles= int(input('What do you have left in soles? '))
dollars= int((pesos*.06)+(reais*.21)+(soles*.28))
print("You have $"+str(dollars)+" left from your trip!")

File: JS Unit/currency.js
// Create a currency converter
let cn = 560; // Chinese yuan
let jp = 2455; // Japanese yen
let kr = 3280; // South Korean won

console.log("What do you have left in yuan? " + cn);
console.log("What do you have left in yen? " + jp);
console.log("What do you have left in won? " + kr);
console.log("");

let dollars = (cn * .14) + (jp * .0068) + (kr * .00075);

console.log("You have $" + dollars + " U.S. dollars.")